### DISCUSSION TITLE
Image Attribution

### DISCUSSION THREAD

**Discussion by davidsj2** [AdaptiveCpp/AdaptiveCpp#964] [General]
Hey,

I'd like to incorporate this image in a slide deck in support of a US government contract: https://github.com/OpenSYCL/OpenSYCL/raw/develop/doc/img/sycl-targets.png

Do I have your permission to include the image with a simple attribution on the slide it is used (e.g. Image Courtesy of <link to image, readme, or github project>)?

**Comment by illuhad** [comment]
Hi!

thanks for asking! yes, the image is under the same license as the rest of the project. I'm fine with any of the variants you have mentioned.

By the way, this image is generated by this script:
https://github.com/OpenSYCL/OpenSYCL/blob/develop/doc/img/render-implementations.py

Feel free to modify it to best fit your slide. There are various display modes that it supports.

**Comment by davidsj2** [comment]
Thanks.  Specifically, I was trying to avoid embedding the full BSD license somewhere in the slide package.

**Reply by illuhad** [reply]
Ah, right. You have permission to not embed the full license and just use a simple attribution as you propose :-)

**Comment by davidsj2** [comment]
Thanks!
